frontend/public/components/operator-lifecycle-manager/create-operand.tsx

```diff
@@ -512,6 +509,8 @@ export const CreateOperand: React.FC<CreateOperandProps> = (props) => {
     .find((s: K8sResourceKind) => referenceFor(s) === referenceForModel(props.operandModel));
   const [method, setMethod] = React.useState<'yaml' | 'form'>('yaml');
 
+  const openAPI = definitionFor(props.operandModel) || _.get(props.customResourceDefinition, ['data', 'spec', 'validation', 'openAPIV3Schema']) as SwaggerDefinition;
```

Comment:
On upstream Kubernetes, CRDs aren't published to the `/openapi/v2` endpoint. So try and use the CRD `spec.validation` block so the form can still work.